presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveFileFormats.java

```diff
@@ -456,6 +465,9 @@ else if (testColumn.getObjectInspector().getTypeName().equals("date")) {
                         SqlDate expectedDate = new SqlDate(((Long) expectedValue).intValue());
                         assertEquals(actualValue, expectedDate, "Wrong value for column " + testColumn.getName());
                     }
+                    else if (testColumn.getObjectInspector().getTypeName().equals("int")) {
```

Comment:
They were needed in checkCursor because they would incorrectly fall into the check for PRIMITIVE. They are not required here.